core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/click/ClickModifier.java

```diff
@@ -50,19 +50,12 @@ public CollectorStepResult collect() throws ProcessingException {
           .waitForElementToBeClickable(webDriver, getLocator(), getTimeoutInSeconds());
       WebElement elementToClick = webDriver.findElement(getLocator());
 
-      if (elementToClick.isDisplayed()) {
-        elementToClick.click();
-        result = CollectorStepResult.newModifierResult();
-      } else {
-        final String message = String
-            .format("Element defined by %s is not yet visible!", getLocator().toString());
-        result = CollectorStepResult.newProcessingErrorResult(message);
-        LOG.warn(message);
-      }
+      elementToClick.click();
+      result = CollectorStepResult.newModifierResult();
     } catch (WebDriverException e) {
       final String message =
-          String.format("No element defined by %s could be found before timeout (%s seconds)!",
-              getLocator().toString(), getTimeoutInSeconds());
+          String.format("Element not found before timeout (%s seconds): %s!",
```

Comment:
I really dislike error messages with the exclamation mark in the end. 
Obviously such sentence should not be provided to the end user - it's part of the development activities. But the communication interface may (should?) stay the same.

I couldn't find any Java standards approach, but as stated here http://stackoverflow.com/a/34136055/774328 the C# documentation explicitely discourages that (https://msdn.microsoft.com/en-us/library/system.exception.message%28v=vs.110%29.aspx) 

> Do not use question marks or exclamation points